src/procedures/SignFreezeIssuanceAck.ts

```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignFreezeIssuanceAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for freezing the issuance of a security token
```

Comment:
Procedure that signs an acknowledgement to permanently freeze issuance of a Security Token